### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/dashboards/about.md [hunk 1]
```diff
@@ -81,8 +81,8 @@ Live mode restrictions do not apply to Dashboard. 
 * A Dashboard can have up to 100 queries.
 * Dashboard chart properties are not retained when viewed from the Search page.
 * Chart properties are not retained when a chart is added to a Dashboard from the Search page.
-* A warning message will be displayed if the number of records returned from the lookup table is more than 1440.
-
+* No dashboard query can return more than 1440 data points. If the dashboard panel has more than 1440 pixels in width, additional resolution in data will be down-sampled to get rendered on the screen.
```
Comment (kimsauce) [inline docs/dashboards/about.md:84]:
```suggestion
* Dashboard queries cannot return more than 1440 data points. If the dashboard panel is wider than 1440 pixels, additional resolution in data will be down-sampled in order to render on the screen.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
one suggestion

File: docs/dashboards/about.md (hunk 1 above)
Comment (oshoghi) [reply docs/dashboards/about.md:84]: @JV0812 please change to `Dashboard queries cannot return more than 1440 data points.` The rest of the explanation is not necessary.

File: docs/dashboards/about.md [hunk 2]
```diff
@@ -81,8 +81,8 @@ Live mode restrictions do not apply to Dashboard. 
 * A Dashboard can have up to 100 queries.
 * Dashboard chart properties are not retained when viewed from the Search page.
 * Chart properties are not retained when a chart is added to a Dashboard from the Search page.
-* A warning message will be displayed if the number of records returned from the lookup table is more than 1440.
-
+* Dashboard queries cannot return more than 1440 data points. If the dashboard panel is wider than 1440 pixels, additional resolution in data will be down-sampled in order to render on the screen.
```
Comment (kimsauce) [inline docs/dashboards/about.md:84]:
```suggestion
* Dashboard queries cannot return more than 1440 data points.
```

**Review by kimsauce — APPROVED** [review]

File: docs/dashboards/about.md (hunk 1 above)
Comment (kimsauce) [reply docs/dashboards/about.md]: @oshoghi , all set! Can you pls approve the PR?

**Pull request merged**